tokio/src/util/linked_list.rs

```diff
@@ -342,22 +295,11 @@ cfg_io_driver_impl! {
 }
 
 cfg_taskdump! {
-    impl<T: Link> CountedLinkedList<T, T::Target> {
-        pub(crate) fn for_each<F>(&mut self, f: F)
-        where
-            F: FnMut(&T::Handle),
-        {
-            self.list.for_each(f)
-        }
-    }
-
     impl<T: Link> LinkedList<T, T::Target> {
-        pub(crate) fn for_each<F>(&mut self, mut f: F)
+        pub(crate) fn for_each<F>(&mut self, f: &mut F)
```

Comment:
Why this change?

Reply:
Yes, it should accept ownership of this `F` closure  instead!

Reply:
Please undo this change and continue to use `mut f: F`. The code in `sharded_list.rs` will still work because mutable references are also `FnMut`.